common.go

```diff
@@ -21,3 +21,22 @@ const (
 	RoleUser      ChatRole = "user"
 	RoleAssistant ChatRole = "assistant"
 )
+
+func translateVertexModel(model Model) string {
```

Comment:
I'd prefer to see this as a method on `Model`, i.e.
```suggestion
func (m Model) asVertex() string {
```

Reply:
Fixed